assets/src/edit-story/components/library/panes/media/common/innerElement.js

```diff
@@ -154,11 +154,10 @@ function InnerElement({
   const imageProps = {
     ...commonImageProps,
     src: thumbnailURL,
-    'aria-label': alt,
+    alt,
   };
   const videoProps = {
     ...commonProps,
-    'aria-label': alt,
```

Comment:
I think we want to keep this here actually, but don't want `alt`.

So

```
'aria-label': alt,
alt: null,
```

Reply:
Okay, I have changed it.